libs/core-ui/src/lib/util/TimeUtils.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+interface ITimeBasedValue {
+  time: number;
+  value: number;
+}
+
+export function orderByTime(values: number[], rowIndices: string[]): any[][] {
+  return values
+    .map((predictedValue: number, idx: number) => {

```

Comment:
this part seems able to be optimized, instead map to obj, sort then map to array, can map to array directly and sort by a[0] - b[0], lodash may even have default behave to sort array by first element, so we can remove call back in sort function

Reply:
Fair! I fixed that.